autosubmitconfigparser/config/configcommon.py

```diff
@@ -851,12 +859,12 @@ def clean_dynamic_variables(self, pattern, in_the_end=False):
 
         for key, dynamic_var in dynamic_variables_.items():
             # if not placeholder in dynamic_var[1], then it is not a dynamic variable
-            if isinstance(dynamic_var[1], list):
-                for value in dynamic_var[1]:
+            if isinstance(dynamic_var, list):
+                for value in dynamic_var:
                     if not re.search(pattern, value, flags=re.IGNORECASE):
                         dynamic_variables[key] = dynamic_var
             else:
-                match = re.search(pattern, dynamic_var[1], flags=re.IGNORECASE)
+                match = re.search(pattern, dynamic_var, flags=re.IGNORECASE)
```

Comment:
See (large) comment to understand the context of this.

Reply:
Good catch! As you commented, it seems that this was an error as a consequence of a bad copy-paste and it had the luck to pass over the tests we made :scream: